servers/scrapping-server/app/product/routes.py

```diff
@@ -7,28 +7,45 @@
 
 @product.route('/product/register/', methods=['POST'])
 def add_product():
-    '''
-    Route to fetch product details for a given url.
-    Method: POST
-    Body: {
-        "url" : ""
-    }
-    Response : JSON object.
-    {
-        'Title': title,
-        'Price': price,
-        'Currency': currency,
-        'MRP': mrp,
-        'Availability': availability,
-        'Rating': rating,
-        'Rating_Count': rating_count,
-        'Image_Link': img_link,
-    } 
-    '''
+    """
+    This function is a Flask route that takes a POST request with a product URL in the body, scrapes the
```

Comment:
Please insert better comment string as it does not describe the route behaviours and parameters

Reply:
i.e it should have request and response parameters as well not only definitions.

Reply:
it's mentioned that "it takes a POST request with a product URL in the body"

Reply:
but it should also mention what should be the key value whether it should be `url` or `product_url` and same applies for the response as from public server we have to know about the whole pattern/format of request as well as response.

Reply:
Fixed,